erpnext_com/www/pricing/faq.html

```diff
@@ -0,0 +1,77 @@
+{% extends "frappe_theme/templates/base.html" %}
+
+{% block content %}
+
+<section class='top-section'>
+	<div class='container'>
+		<h1>Frequently Asked Questions</h1>
+		<div class='text-left'>
+			<p class="mt-5">
+				<h5>How is the fees calculated for the number of users?</h5>
+				You only pay for the number of System users in your accounts. These are the users who can access the core modules of the ERP and not just portal views.<br><br> 
+                
+                A system user is generally someone with system roles like manager in Sales/Purchase/Manufacturing/HR, Leave Approver, Employee, Auditor, etc. These roles are assigned from the user's settings page.
+			</p>
+			<p class="mt-5">
+                <h5>What am I paying for here?</h5>
+                <ul>
+                    <li>Hosting, backups, and storage managed by us.</li>
+                    <li>Priority functional support by the team that made ERPNext.</li>
+                    <li>Your own subdomain (with a choice of accessing via your custom domain).</li>
+                    <li>Bug Fixing guarantee.</li>
+                </ul>
+			</p>
+			<p class="mt-5">
+				<h5>I want to host ERPNext on my own server but need your support. Is this possible?</h5>
+				Yes! We do provide support for self-hosted instances as well. Check out our <a href="https://erpnext.com/support">support plan</a> for more details.
+			</p>
+			<p class="mt-5">
+				<h5>How to proceed with the subscription payment?</h5>
+                You can pay for your ERPNext account from the application itself! Go to <b>My Account > Upgrade</b>.
+                <div> <!--class="col-sm-12 text-center mar-t-40"> -->
+                    <img src="/assets/erpnext_com/images/subscribe.gif" class="img-responsive"
```

Comment:
the GIFs here represent v11, can a gif on v12 be made?

Reply:
There is no cloud account on v12 so far. Hence can't make a gif of the same